src/ApiService/ApiService/onefuzzlib/orm/Queries.cs

```diff
@@ -11,6 +11,9 @@ public static class Query {
         public static string PartitionKey(string partitionKey)
             => TableClient.CreateQueryFilter($"PartitionKey eq {partitionKey}");
 
+        public static string PartitionKey(Guid partitionKey)
+            => TableClient.CreateQueryFilter($"PartitionKey eq {partitionKey}");
+
```

Comment:
What about something like:
```csharp
public static string PartitionKey<T>(T partitionKey)
            where T : IFormattable
            => TableClient.CreateQueryFilter($"PartitionKey eq {partitionKey}");
```

That way we can also add it to `PoolName` for example.

Reply:
Also had this idea but hadn’t gotten around to it 👍